cl/search/admin.py

```diff
@@ -259,6 +296,28 @@ class DocketAdmin(CursorPaginatorAdmin):
         "parent_docket",
     )
 
+    def change_view(self, request, object_id, form_url="", extra_context=None):
+        """Add links to pre-filtered related admin pages."""
+        extra_context = extra_context or {}
+        docket = self.get_object(request, object_id)
```

Comment:
Is it possible to avoid this docket lookup?
I believe it's possible to omit it by removing the if statements below:
`if docket and hasattr(docket, "docket_entries")`
and
`if docket and hasattr(docket, "alerts")`

And just build the URLs using the `object_id` since in case the `docket_id` doesn't exist, the detail admin will return a 404 error, so perhaps these safety checks can be omitted?